tests/layer/test_hinsage.py

```diff
@@ -270,41 +270,6 @@ def test_hinsage_apply():
     assert actual == pytest.approx(expected)
 
 
-def test_hinsage_default_model():
```

Comment:
Why delete this test? Despite the name, it was using `build`?

Reply:
ah I was too hasty to delete this one, I'll ad it back and rename it

Reply:
@kjun9 (easier to reply to one thread)
actually I think this test is a duplicate of `test_hinsage_apply` now that we've removed `default_model`